src/Stratis.FederatedPeg.Features.FederationGateway/FederationGatewaySettings.cs

```diff
@@ -76,6 +79,9 @@ public FederationGatewaySettings(NodeSettings nodeSettings)
         /// <inheritdoc/>
         public int MultiSigN { get; }
 
+        /// <inheritdoc/>

```

Comment:
This isn't used anywhere apart from picking up the value from config.

Reply:
Same with `MultiSigN`. Let's hold on to `MinCoinMaturity` until we decide if it should be configurable. We may need to have it set to 0 to ensure that change outputs can immediately be spent by subsequent deposits.

Reply:
Ok.